applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua

```diff
@@ -67,9 +67,11 @@ function s.create(self, name)
 		uci:save("openvpn")
 
 		luci.http.redirect( self.extedit:format(name) )
-	else
+	elseif string.len(name)>0 then
```

Comment:
Please use `#name > 0` here

Reply:
No problem, but I have used string.len() because it's already used some lines above:
https://github.com/openwrt/luci/blob/master/applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua#L59

Should I then change this too?

Reply:
@jow- Whats your opinion?

Reply:
@sch-m in this case yes...